pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Producer.java

```diff
@@ -852,4 +852,25 @@ public boolean isDisconnecting() {
 
     private static final Logger log = LoggerFactory.getLogger(Producer.class);
 
+    /**
+     * Increments the counter that controls the throttling of the connection by pausing reads.
```

Comment:
**[Nit]** This sounds like it increments the counter by pausing reads but really it just increments the counter and the counter is described as... (same for decrement method). I think something like this may clarify it a bit, wdyt?

```
    /**
     * Increments the counter that controls the throttling of the connection.
     * <p>
     * When the counter is greater than 0, the connection is throttled by pausing reads.
     * <p>
     * The caller is responsible for decrementing the counter by calling {@link #decrementThrottleCount()}  when the
     * connection should no longer be throttled.
     */
```

Reply:
I made improvements to the javadoc. PTAL